dashboard/config/environments/test.rb

```diff
@@ -29,6 +29,10 @@
 
     # Version of your assets, change this if you want to expire all your assets.
     config.assets.version = '1.0'
+
+    # Avoid loading all i18n files up front, which can significantly slow down initialization.
+    # Instead, it only loads i18n files that belong to the current locale.
+    config.i18n.backend = Cdo::I18n::LazyLoadableBackend.new(lazy_load: true)
```

Comment:
We use it only in the local test environment, so why not use it for integration tests too?

Reply:
In Ruby on Rails, the test environment is specifically tailored for running tests.
I don't know if we're using it for that website, but it would be incorrect to do so.